pkg/operator/controllers/gameserverbuild_controller.go

```diff
@@ -149,16 +158,24 @@ func (r *GameServerBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 			initializingCount++
 		} else if gs.Status.State == mpsv1alpha1.GameServerStateStandingBy && gs.Status.Health == mpsv1alpha1.GameServerHealthy {
 			standingByCount++
+			if gs.Status.State != gs.Status.PrevState {
+				timeToStandBySum += getStateDuration(gs.Status.ReachedStandingByOn, &gs.CreationTimestamp)
+				recentStandingByCount++
+			}
 		} else if gs.Status.State == mpsv1alpha1.GameServerStateActive && gs.Status.Health == mpsv1alpha1.GameServerHealthy {
 			activeCount++
 		} else if gs.Status.State == mpsv1alpha1.GameServerStateGameCompleted && gs.Status.Health == mpsv1alpha1.GameServerHealthy {
 			// game server process exited with code 0
 			if err := r.Delete(ctx, &gs); err != nil {
 				return ctrl.Result{}, err
 			}
+
 			GameServersSessionEndedCounter.WithLabelValues(gsb.Name).Inc()
 			r.expectations.addGameServerToUnderDeletionMap(gsb.Name, gs.Name)
 			r.Recorder.Eventf(&gsb, corev1.EventTypeNormal, "Exited", "GameServer %s session completed", gs.Name)
+
+			pendingCleanUpCount++
+			timeToDeleteSum += getStateDuration(gs.DeletionTimestamp, &gs.CreationTimestamp)
```

Comment:
Is the DeletionTimeStamp set with the current upstream code in `main` branch? We removed the finalizer (exists in 0.5.0) so I assume that this is not set at all now.

Reply:
https://kubernetes.io/blog/2021/05/14/using-finalizers-to-control-deletion/

Reply:
Scale down metrics were removed